app/Http/Controllers/AdminFoodController.php

```diff
@@ -52,4 +74,13 @@ public function addFoodSubmit(Request $request)
 
         return redirect()->back()->with('msg-ok', __('msg.add_ok', ['name' => $request->get('name')]));
     }
+
+    public function foodsList()
+    {
+        allowed(Role::FOOD_MANAGER);
+
+        $data['foods'] = Food::query()->orderBy('restaurant_id')->get();
```

Comment:
please use Eager Loading

Reply:
there is no relation here ! for eager loading

Reply:
done